### ISSUE TITLE
.plan 2018-05-08

### ISSUE THREAD

**Issue by geewiz** [geewiz/dev0ps#13]
- [x] Set up Gitlab on Kontena

**Comment by int3l** [comment]
I managed to bring the gitlab container up. You should consider that the minimum RAM requirement is around 4GB for the default image. It took about 1-2 minutes (on SSD) to bootstrap the first time.
So check the Kontena resource pool in order to be sure, that there is no bottleneck.

Also you should follow the container/service logs, because the install procedure indicates when the bootstrap process completes.

BTW: I managed to reduce the RAM usage to around 2GB, which is the recommended minimum for up to 100 users (effectively 50 users for smooth service experience). But it requires adding additional configs to the OMNIBUS config section or direct injection in the gitlab.rb config file.

**Comment by geewiz** [comment]
@int3l Thanks for the follow-up! You're spot on, it was the memory requirements. Man, I should have checked my Github notifications earlier. 😊 (In my defense, I've taken this week off and try not to check my email.)

Yesterday, I tried a local `docker-compose up`, which worked fine. Then I confirmed with the Kontena Slack that my Gitlab stack config was correct. So this morning I checked the infrastructure and it turned out I still had Kontena nodes with 2GB from my initial tests. Upgraded the RAM, issue resolved. Damn!

Well, it's still a valuable lesson about Kontena ops basics.
